scripts/gasCompare.js

```diff
@@ -79,21 +76,15 @@ let gasCompare = async () => {
             }
         }
     }
-let markdownstring = markdown(gasChangeData,coloumn);
-if(gasChangeData.length!==0){
-    
-        await commenter.createOrUpdateComment('gasCompare', markdownstring ).catch(err=>{  
-                console.log(markdownstring);
-        })
+
+    let markdownString = markdown(gasChangeData, coloumn);
+
+    if (gasChangeData.length !== 0) {
+        // Write the markdown to a file called gasCompareOutput.json
```

Comment:
Write the gas compare output to the console as well. This way we can at least see the gas compare in the actions logs.

Reply:
Okay, will note this on the gas compare issue and tackle it then. Works ? yohan/raz-416-gascompare-needs-to-be-added-to-github-actions @SkandaBhat

Reply:
can you confirm @SkandaBhat, since we will not be running gasCompare on the CI temporarily.